shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -211,9 +211,15 @@ void PostAccessibilityNotification(UIAccessibilityNotifications notification,
     }
 
     if (layoutChanged) {
+      SemanticsObject* next = FindNextFocusableIfNecessary();
+      SemanticsObject* lastFocused =
+          [objects_.get() objectForKey:@(last_focused_semantics_object_id_)];
+      // Only specify the focus item if the new focus is different, avoiding double focuses on the
+      // same item. See: https://github.com/flutter/flutter/issues/104176. If there is a route
+      // change, we always refocus.
       ios_delegate_->PostAccessibilityNotification(
           UIAccessibilityLayoutChangedNotification,
-          FindNextFocusableIfNecessary().nativeAccessibility);
+          (routeChanged || next != lastFocused) ? next : NULL);
```

Comment:
Should it be `next.nativeAccessibility` instead of `next`?

Reply:
Fixed, ptal